src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -128,6 +135,31 @@ def close(self):
         # Constructor exceptions may occur prior to _session being set.
         if hasattr(self, '_session'):
             self._session.close()
+        if hasattr(self, '_https_session'):
+            self._https_session.clear()
+
+    @staticmethod
+    def make_https_session(ca_certs, certs, auth, key_password):
```

Comment:
`ca_certs_path`

Reply:
Use two different parameters for certificate and key. Don't use `certs` to represent both.

Reply:
Done !